### ISSUE TITLE
.slim-lint.yml is not read

### ISSUE THREAD

**Issue by crispinheneise** [aliariff/vscode-slim-lint#1]
Slim-lint is supposed to [read a configuration file](https://github.com/sds/slim-lint#configuration).

I have a `.slim-lint.yml` config file in my project folder, but vscode-slim-lint doesn't seem to be picking it up. 

<img width="1198" alt="image" src="https://user-images.githubusercontent.com/234/52449378-bcd96600-2afc-11e9-9edc-f045d4bb7284.png">

```
{
	"resource": "/path/to/project/app/views/people/registrations/new.html.slim",
	"owner": "slim-lint",
	"severity": 4,
	"message": "LineLength: Line is too long. [118/80]",
	"startLineNumber": 11,
	"startColumn": 1,
	"endLineNumber": 11,
	"endColumn": 119
}
```

**Comment by aliariff** [comment]
@crispinheneise 

Hi thanks for the feedback, yes currently is not supported to read the config file.
I will update it ASAP, or maybe if you can, create a pull request for that.

**Comment by aliariff** [comment]
@crispinheneise 

Fix in version 0e3754bffb1ddc937f02f615731bc9bee40ce256

**Comment by crispinheneise** [comment]
Well, you beat me to it! Thanks!
